src/test/java/edu/iis/powp/drawer/TestDrawer.java

```diff
@@ -19,7 +19,7 @@ public static void main(String[] args) {
 		DefaultDrawerFrame.getDefaultDrawerFrame().setVisible(true);
 		ILine line = LineFactory.getBasicLine();
 		line.setStartCoordinates(-100, -60);
-		line.setEndCoordinates(60, 130);
+		line.setEndCoordinates(60, 10);

```

Comment:
Is committing those changes necessary? It makes tracking actual changes more difficult.